### PR TITLE
fix: record localdebug status

### PR DESCRIPTION
**Pull request by xuruiyao-msft**
Record the local debug success/failure in .fx/projectStatus.json. It would be used in the nextStep for Office GitHub Agent project.

### CODE REVIEW

File: packages/vscode-extension/src/debug/officeTaskHandler.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        const debugConfig = event.configuration;
+        if (
+            debugConfig &&
+            debugConfig.name &&
+            !debugConfig.postDebugTask
+        ) {
+            if (await commonUtils.checkAndSkipDebugging()) {
+                throw new Error(DebugSessionExists);
+            } else {
+                commonUtils.startLocalDebugSession();
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/officeTaskHandler.ts:113]: Will `startLocalDebugSession` be called twice? One from `teamsfxTaskHandler` and another from here.
I would suggest to totally separate the debug session handler from the original teamsfx one, e.g., record office addin's own sessions here.

File: packages/vscode-extension/src/debug/officeTaskHandler.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
+import { ok } from "@microsoft/teamsfx-api";
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/officeTaskHandler.ts:1]: Copyright header.
And also adjust the code owner in `.github/CODEOWNERS`

File: packages/vscode-extension/src/debug/officeTaskHandler.ts (hunk 1 above)
Comment (xuruiyao-msft) [reply packages/vscode-extension/src/debug/officeTaskHandler.ts:113]: @swatDong  Thanks for your comments. I went through the extension.ts activate function :

export async function activate(context: vscode.ExtensionContext) {
  ........
  if (isTeamsFxProject) {
    activateTeamsFxRegistration(context);
  }

  if (isOfficeAddInProject) {
    activateOfficeDevRegistration(context);
  }
  ......
  }
The teamsfxTaskHandler is registered in activateTeamsFxRegistration, but the officeTaskHandler is registered in activateOfficeDevRegistration. They are mutually exclusive for each other. startLocalDebugSession will not be called twice here.

Comment (swatDong) [reply packages/vscode-extension/src/debug/officeTaskHandler.ts:113]: Thanks for the explanation, make sense to me.

File: packages/vscode-extension/src/debug/officeTaskHandler.ts (hunk 2 above)
Comment (xuruiyao-msft) [reply packages/vscode-extension/src/debug/officeTaskHandler.ts:4]: Thanks for your comments. Updated in the latest iteration. Would you please help review it again? Thanks for your effort.

**Review by swatDong — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Review by MSFT-yiz — APPROVED** [review]

**Pull request merged**